Documentation/Installation/SystemRequirements/Index.rst

```diff
@@ -8,7 +8,23 @@
 System requirements
 ===================
 
-For up-to-date information about TYPO3's system requirements visit `get.typo3.org
+:Operating System:
```

Comment:
I think we should not copy the system requirements here, they can be reviewed at get.typo3.org. I think we should rather put some general explainations like "You need a webserver with a supported TYPO3 version, see here which versions are currently supported" or some-such